plugins/bean-validation/src/main/java/org/apache/struts/beanvalidation/constraints/ValidateGroup.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.struts.beanvalidation.constraints;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Validation with Group Constrains on execution method (Action)
+ * <p>
+ * Example, Validate with on special group
+ * </p>
+ * <pre>
+ *
+ *  {@literal @}ValidateGroup(CarChecks.class)
+ *  {@literal @}Action...
+ * </pre>
+ * <p>
+ * <p>
+ * Example, Validate with severals special group
+ * </p>
+ * <pre>
+ *  {@literal @}ValidateGroup(Default.class, CarChecks.class, DriverChecks.class)
+ *  {@literal @}Action...
+ * </pre>
+ */
+@Target({ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ValidateGroup {
+
+    Class<?>[] value() default {};
```

Comment:
I would put `Default.class` as a default value, in such case specifying an empty annotation won't break anything.

Reply:
if i add the Default.class to the default value in the Annotation (ValidationGroup), it becomes a depencies to the javax.validation package. Is this a good practise?

Reply:
but this plugin already depends on that package so it won't be a problem :)